Expand example content for extended markdown usage (#2819)
Kong/docs.konghq.com: app/enterprise/2.3.x/developer-portal/theme-customization/markdown-extended.md

## Specify the markdown rendering module in a document

> 1. Create a markdown file for your Dev Portal documentation. The markdown file, for example `markdown-example.md` needs to be added under `content` folder. Add your markdown content, refer [Markdown Cheat Sheet](https://www.markdownguide.org/cheat-sheet/) to start with. The example below has few content for your references.

Reviewer: ```suggestion
1. Create a markdown file for your Dev Portal documentation. The markdown file, for example `markdown-example.md`, needs to be added under the `content` folder. Add your markdown content. Refer to the [Markdown Cheat Sheet](https://www.markdownguide.org/cheat-sheet/). The example below shows some basic markdown syntax.
```
